### PR TITLE
Added support for building Amazon Linux 2 package

### PR DESCRIPTION
**Pull request by alula-mhayes**

### CODE REVIEW

File: rpm/awsbuild.sh [hunk 1]
```diff
@@ -1,12 +1,28 @@
...
+
+build2() {
+  rpmbuild_opts="--define '_topdir /mnt/build/rpmbuild' -bb rpm/netns-auto-cfg-2.spec"
+
+  docker pull amazonlinux:2 || exit 1
+  docker run \
+    --rm \
+    --mount type=bind,source=$(pwd),destination=/mnt/build \
+    amazonlinux:2 \
+    bash -c "yum install -y rpmdevtools make && cd /mnt/build && rpmbuild $rpmbuild_opts"
```
Comment (jonjaques) [inline rpm/awsbuild.sh]: damn, this is the only thing preventing passing a param 😸 to build()

Comment (alula-mhayes) [reply rpm/awsbuild.sh]: tbh i should just remove `build1()`, since there's no reason we'd go back to amazon linux 1.

Comment (alula-mhayes) [reply rpm/awsbuild.sh]: i'll just go ahead and do that to avoid confusion

Comment (alula-mhayes) [reply rpm/awsbuild.sh]: pushed commit for that

**Pull request merged**